### PR TITLE
CANS-908 Added a placeholder in Client Search

### PR DESCRIPTION
**Pull request by pokharelmanish**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-908

## Description
Added placeholder in Client Search.

Before:
screen shot 2018-12-28 at 11 26 19 am

After:
screen shot 2018-12-28 at 11 26 06 am

## Tests

- [x] I used TDD to develop the feature
- [x] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Search/Autocomplete/style.sass [hunk 1]
```diff
@@ -123,4 +123,12 @@
...
\ No newline at end of file
+
+//********************
+//Client Search
+//********************
+input[id='client-search-autocompleter']::placeholder
+  font-family: 'Source Sans Pro'
+  font-size: 1.5rem
+  font-weight: 400
+  color: $gray29
```
Comment (pokharelmanish) [inline app/javascript/Application/components/Search/Autocomplete/style.sass]: This are all the attributes provided by the design team.

**Comment by pokharelmanish** [comment]
retest this please

File: app/javascript/Application/components/Search/Autocomplete/style.sass (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/components/Search/Autocomplete/style.sass]: why are we not using the colors and constants here?

Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/components/Search/Autocomplete/style.sass]: why are we not using the colors, constants that are already defined in styles?

Comment (NagaPradeepGajula-osi) [reply app/javascript/Application/components/Search/Autocomplete/style.sass]: ok just noticed that there are no constants defined in global styles

Comment (pokharelmanish) [reply app/javascript/Application/components/Search/Autocomplete/style.sass]: $gray29 is a constant

**Comment by pokharelmanish** [comment]
Due to https://github.com/ca-cwds/cans/pull/393 I need to redo the work so I am going to close this PR and create a new one. This work is pretty straight forward.

**Pull request closed without merging**